torque-core/src/main/kotlin/com/workday/torque/TestRunFactory.kt

```diff
@@ -92,6 +94,17 @@ class TestRunFactory {
         }
     }
 
+    private fun getTimeoutMillis(args: Args, installer: Installer, testChunk: TestChunk): Long {
+        val chunkTimeOutWithRetries = TimeUnit.SECONDS.toMillis(args.chunkTimeoutSeconds) * args.retriesPerChunk
+        val installTimeOutWithRetries = TimeUnit.SECONDS.toMillis(args.installTimeoutSeconds.toLong()) * args.retriesInstallPerApk
```

Comment:
Should this be moved into the else? Only needs to be calculated in that case

Reply:
It's a trivial calculation and I think it reads better with them grouped up here.

Reply:
Both works, but if-else brackets :P?

Reply:
fun result, install retries happen within each chunk retry.
So it's actually 
```
val installTimeOutWithRetries = TimeUnit.SECONDS.toMillis(args.installTimeoutSeconds.toLong()) * args.retriesPerChunk * args.retriesInstallPerApk
```